src/features/stats/index.js

```diff
@@ -38,9 +39,33 @@ class Stats extends Component {
 
     render() {
         const { disabled, stats, sideMenu, largeView } = this.props;
-        const { abilities, level, exp, expToLevel, hp, maxHp, gold } = stats;
+        const {
+            abilities,
+            level,
+            exp,
+            expToLevel,
+            hp,
+            maxHp,
+            mana,
+            maxMana,
+            gold,
+            start,
```

Comment:
This assignment of `start` produces a warning since the value isn't used.

Reply:
_start_ has been removed now.